webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java

```diff
@@ -54,6 +54,21 @@ public class NamespaceFileController {
     );
 
 
+    @ExecuteOn(TaskExecutors.IO)
+    @Get(uri = "{namespace}/files/search", produces = MediaType.TEXT_JSON)
+    @Operation(tags = {"Files"}, summary = "Find files which path contain the given string in their URI")
+    public List<String> search(
+        @Parameter(description = "The namespace id") @PathVariable String namespace,
+        @Parameter(description = "The string the file path should contain") @QueryValue String q
+    ) throws IOException, URISyntaxException {
+        URI baseNamespaceFilesUri = toNamespacedStorageUri(namespace, null);
+        return Stream.concat(
```

Comment:
Why do you concat with staticFiles and what is this statifFiles?

Reply:
For easier maintainability and flexibility I added a static file concept for VSCode editor. Those files are served from static resources of the webserver and the goal is to be able as a software to provide some read-only files to the user. This can be used to guide him (for eg. we currently have a getting-started file which appear on VSCode editor but isn't editable, it is served statically and acts like a README for editor usage)

Reply:
Well, I'm not sure I see the point of including them in the search but anyway, OK